web/ajax/stream.php

```diff
@@ -21,23 +21,23 @@
 switch ( $_REQUEST['command'] )
 {
     case CMD_VARPLAY :
-        Debug( "Varplaying to ".$_REQUEST['rate'] );
+        Logger::Debug( "Varplaying to ".$_REQUEST['rate'] );
```

Comment:
Why put the Logger:: in front?  It's outside the scope of the PR.  I actually do like to fully specify function names, but this seems unnecessary.